File: frontend/src/app/modules/cattle-sales/domain/dto/create-cattle-sale-item.dto.ts
export interface CreateCattleSaleItemDto {
  tipoOrigen: 'GANADO' | 'TERNERO';
  idOrigen: number;
  pesoVenta: number;
  precioKilo: number;
  precioTotal: number;

  numero_ganado?: string | number; // Opcional, solo si tipoOrigen es 'GANADO'
}

File: backend/src/main/java/com/erp/backend_erp/dto/cattleSales/CreateCattleSaleItemDto.java
package com.erp.backend_erp.dto.cattleSales;

import java.math.BigDecimal;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class CreateCattleSaleItemDto {
    private String numero_ganado;
    private String tipoOrigen;
    private Long idOrigen;
    private BigDecimal pesoVenta;
    private BigDecimal precioKilo;
    private BigDecimal precioTotal;
}
